packages/flutter/lib/src/material/dialog.dart

```diff
@@ -60,12 +61,33 @@ class Dialog extends StatelessWidget {
   /// Defaults to [Curves.fastOutSlowIn].
   final Curve insetAnimationCurve;
 
+  /// The semantic label of the dialog used by accessibility frameworks to 
+  /// announce screen transitions when the dialog is opened and closed.
+  /// 
+  /// If this label is not provided, it will default to
+  /// [MaterialLocalizations.dialogLabel].
```

Comment:
Did you add the localizations in a separate PR? Or should they be part of this PR?

Reply:
I added them in a separate PR -https://github.com/flutter/flutter/pull/16633